### PR TITLE
feat(checker): Do not output anything if there are no conflicting rules

### LINKED ISSUE #133: Silence tslint-config-prettier-check on success
Linting tools (such as TSLint or ESLint) often do not output anything by default when there is nothing to report. I have integrated `tslint-config-prettier-check` in my linting workflow which is really nice to make sure no broken configurations sneak in:

```
$ yarn lint
yarn run v1.5.1
$ run-s lint:* tslint-check
$ tslint --format stylish bin/**/*.ts
$ tslint --format stylish --project .
$ tslint-config-prettier-check ./tslint.yml
No conflict rule detected in ./tslint.yml
✨  Done in 10.76s.
```

It would be nice if the default case would not return anything as well. I don't think it's worth adding a `--verbose` flag (and potential dependency) to get the current way back, but you tell me :)
I'll open a PR in case you find this reasonable, but if you don't like the idea, I'll understand!

### PR DESCRIPTION
**Pull request by astorije**
Resolves #133.

### CODE REVIEW

File: tools/__snapshots__/checker.test.ts.snap [hunk 1]
```diff
@@ -24,13 +24,12 @@ exports[`should be able to check multiple config files 1`] = `
...
 [error]   whitespace
-[log] No conflict rule detected in <cwd>/fixtures/tslint.empty.json
 [error] Unexpected number in JSON at position 1 in <cwd>/fixtures/tslint.error.json"
 `;
 
-exports[`should show success message for conflict rules that is disabled 1`] = `"[log] No conflict rule detected in <cwd>/fixtures/tslint.false.json"`;
+exports[`should show success message for conflict rules that is disabled 1`] = `""`;
 
-exports[`should show success message for non-conflict rules 1`] = `"[log] No conflict rule detected in <cwd>/fixtures/tslint.empty.json"`;
+exports[`should show success message for non-conflict rules 1`] = `""`;
```
Comment (ikatyang) [inline tools/__snapshots__/checker.test.ts.snap]: We should also update their titles.

Comment (astorije) [reply tools/__snapshots__/checker.test.ts.snap]: Ah yes, I messed that up and forgot. Done now, thanks!

**Comment by astorije** [comment]
Yay, awesome thanks! 🎉
Can't wait for the next release now :)

**Comment by astorije** [comment]
That was so fast! Thanks @ikatyang :)

**Pull request merged**